### DISCUSSION TITLE
32

### DISCUSSION THREAD

**Discussion by giscus** [car-ffeine/car-ffeine.github.io#17] [Announcements]
# 32

먼저 이 글을 쓰게 된 계기를 말씀드리겠습니다. 지난 글에서 설명했듯이 저희 프로젝트에서는 데이터베이스가 실행되고 있는 서버의 cpu 사용률이 100%가 되는 문제가 있었습니다.

https://car-ffeine.github.io/32

**Comment by greeng00se** [comment]
replication 적용하고 사이트가 엄청 빨라졌던데 대단합니다!
